.github/workflows/builder.yml

```diff
@@ -83,14 +83,14 @@ jobs:
       - name: Check add-on
         id: check
         run: |
-           if [[ "${{ steps.info.outputs.architectures }}" =~ ${{ matrix.arch }} ]]; then
-              echo "::set-output name=build_arch::true";
-            else
-              echo "${{ matrix.arch }} is not a valid arch for ${{ matrix.addon }}, skipping build";
-           fi
+          if [[ "${{ steps.info.outputs.architectures }}" =~ ${{ matrix.arch }} ]]; then
+             echo "::set-output name=build_arch::true";
+           else
+             echo "${{ matrix.arch }} is not a valid arch for ${{ matrix.addon }}, skipping build";
+          fi
 
-           version=$(jq -r '.version' "./${{ matrix.addon }}/config.json")
-           echo "::set-output name=version::${version}";
+          version=$(jq -r '.version' "./${{ matrix.addon }}/config.json")
```

Comment:
```suggestion
          version=$(jq -r '.version' "./${{ matrix.addon }}/config.yaml")
```

Reply:
Line 33 also needs the same update, but can't suggest it.

Reply:
We are not using `steps.check.outputs.version` so those 2 version related lines can be removed

Reply:
Line 33 still needs to be changed from  `.json` -> `.yaml`